.github/workflows/build-test.yml

```diff
@@ -340,234 +340,6 @@ jobs:
           path: .nyc_output
           retention-days: 1
 
-  tests:
```

Comment:
All of the tests have been commented out, hence absolutely no point in running a job that does nothing.

Reply:
I.e. none of the functional tests work today (all have been commneted after webview was removed)